swagger_parser/lib/src/parser/utils/type_utils.dart

```diff
@@ -213,3 +213,7 @@ final _nameRegExp = RegExp(r'^[a-zA-Z_][a-zA-Z\d_]*$');
     },
   );
 }
+
```

Comment:
Add docs like in functions above
```
/// Protect name from incorrect symbols, keywords, etc.
```

Reply:
It had one before the merge conflict 😊
Done